README.md

```diff
@@ -395,6 +397,34 @@ If you want to use [Relay](https://facebook.github.io/relay/) together with this
 
 Please read our [guidelines](.github/CONTRIBUTING.md).
 
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](.github/CONTRIBUTING.md)].
```

Comment:
@monkeywithacupcake This line still has double square brackets. If it's not too much trouble it would be nice if you could fix that so that we can get this merged.

Reply:
Also, we could remove the **Contributing** section in favor of the **Contributors** section you added. Now they both link to the same document.